eox_hooks/actions_handler.py

```diff
@@ -18,9 +22,9 @@ def action_handler(trigget_event, configuration, **kwargs):
 
     try:
         action(**kwargs)
-        log.info("The action {} with triggered by {} ended successfully.".format(action, trigget_event))
+        audit_action_execution(Status.SUCCESS, trigget_event, action)
     except Exception as exception:  # pylint: disable=broad-except
-        log.error("The action {} with triggered by {} failed.".format(action, trigget_event))
+        audit_action_execution(Status.FAIL, trigget_event, action)
```

Comment:
do we want to write to the audit model if an Exception was raised, and the message from said exception?

Reply:
That would be helpful and it would give more context!